### PR TITLE
add logic

### PR DESCRIPTION
**Pull request by BelehovEgor**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/635

### CODE REVIEW

File: src/UserService/Startup.cs [hunk 1]
```diff
@@ -266,7 +266,7 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
 
             app.UseRouting();
 
-            app.UseMiddleware<TokenMiddleware>();
```
Comment (TIove) [inline src/UserService/Startup.cs:269]: .

**Review by TIove — APPROVED** [review]

File: src/UserService.Business/Commands/User/CreateUserCommand.cs [hunk 2]
```diff
@@ -42,14 +42,61 @@ public class CreateUserCommand : ICreateUserCommand
...
+                    _logger.LogWarning(
+                        "Сan't assign user {userId} to the department {departmentId}. Please try again later.",
+                        userId, departmentId);
+
+                    errors.Add(errorMessage);
+                }
+            }
+            catch (Exception exc)
+            {
+                _logger.LogWarning(
```
Comment (djino97) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:65]: Use LogError

File: src/UserService.Business/Commands/User/CreateUserCommand.cs [hunk 3]
```diff
@@ -42,14 +42,61 @@ public class CreateUserCommand : ICreateUserCommand
...
-            // TODO add user department
+            string errorMessage = $"Сan't assign user {userId} to the department {departmentId}. Please try again later.";
+
+            try
+            {
+                var response = _rcDepartment.GetResponse<IOperationResult<bool>>(
+                    IChangeUserDepartmentRequest.CreateObj(userId, departmentId)).Result;
+                if (!response.Message.IsSuccess || !response.Message.Body)
+                {
+                    _logger.LogWarning(
```
Comment (djino97) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:56]: Why not use errorMessage  variable? And no use response.Message.Errors?

File: src/UserService/Startup.cs [hunk 4]
```diff
@@ -266,7 +266,7 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
...
+            //app.UseMiddleware<TokenMiddleware>();
```
Comment (djino97) [inline src/UserService/Startup.cs:269]: need uncomment

**Review by djino97 — APPROVED** [review]

File: src/UserService/Startup.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/UserService/Startup.cs:269]: fixed

File: src/UserService.Business/Commands/User/CreateUserCommand.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/UserService.Business/Commands/User/CreateUserCommand.cs:65]: fixed

File: src/UserService/Startup.cs (hunk 4 above)
Comment (BelehovEgor) [reply src/UserService/Startup.cs:269]: fixed

File: src/UserService.Business/Commands/User/CreateUserCommand.cs (hunk 3 above)
Comment (BelehovEgor) [reply src/UserService.Business/Commands/User/CreateUserCommand.cs:56]: this var uses $, but we don't need it in logging

**Review by MariyaSin — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

**Pull request merged**